runtime-watcher/go.mod

```diff
@@ -5,78 +5,44 @@ go 1.24.6
 require (
 	github.com/go-logr/logr v1.4.3
 	github.com/go-logr/zapr v1.3.0
-	github.com/google/uuid v1.6.0
-	github.com/kyma-project/lifecycle-manager/api v0.0.0-20250825111725-e7360b26f3df
-	github.com/kyma-project/runtime-watcher/listener v0.0.0-20250825111833-2cf3f8cc5232
-	github.com/onsi/ginkgo/v2 v2.25.3
-	github.com/onsi/gomega v1.38.2
+	github.com/kyma-project/runtime-watcher/listener v0.0.0-20250908061848-5ecf98a16e6b
```

Comment:
Is it the latest version of the listener? After merge of this PR which also has no controller-runtime dependencies
https://github.com/kyma-project/runtime-watcher/pull/616